.buildkite/heartbeat/scripts/integration-pytests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/env-scripts/linux-env.sh
+
+echo "--- Executing Integration Tests"
+sudo chmod -R go-w heartbeat/
```

Comment:
also leave a comment here that this will like not be needed once we start using a vm that has the right umask harded coded (which will result in not having `w` privileges for `go` for the checked out directory)